src/cljs/commiteth/common.cljs

```diff
@@ -12,16 +12,24 @@
                    :on-change #(reset! val-ratom (-> % .-target .-value))})]))
 
 (defn dropdown [props title val-ratom items]
-  (fn []
-    (if (= 1 (count items))
-      (reset! val-ratom (first items)))
-    [:select.ui.basic.selection.dropdown
-     (merge props {:on-change
-                   #(reset! val-ratom (-> % .-target .-value))})
-     (doall (for [item items]
-              ^{:key item} [:option
-                            {:value item}
-                            item]))]))
+  "If val-ratom is set, preselect it in the dropdown.
+   Add value of val-ratom if it's missing from items list.
+   Otherwise, prepend title as a disabled option"
+  (let [items (cond-> items
+                  (and @val-ratom
+                       (not (contains? (set items) @val-ratom)))
```

Comment:
Like the use of `contains?` with a set instead of just using the set as a function, since it is much clearer! :)